### ISSUE TITLE
[Bug]: cgdb: fdsan issue

### ISSUE THREAD

**Issue by easyaspi314** [termux/termux-packages#7881]
### Problem description

`cgdb` fails to run on Android 11 due to `fdsan`.
```
fdsan: failed to exchange ownership of file descriptor: fd 6 is owned by FILE* 0x7eedebf578,
was expected to be unowned
```

### What steps will reproduce the bug?

Try to run cgdb.

Except for the help/version messages, everything seems to immediately crash.

While I haven't tested or compiled a debug build, the only calls to `fdopen` are two sequential ones which use the same file descriptor for input and output. The strace I did see closely matches it.
 https://github.com/cgdb/cgdb/blob/f9e2461944ae2d2123dd2b3d1140fe74fd08b3cb/lib/rline/rline.cpp#L104 

### System information

termux-info:

```
Application version:
0.117
Packages CPU architecture:
aarch64
Subscribed repositories:
# sources.list
deb https://packages.termux.org/apt/termux-main/ stable main
# game-repo (sources.list.d/game.list)
deb https://packages.termux.org/apt/termux-games games stable
# science-repo (sources.list.d/science.list)
deb https://packages.termux.org/apt/termux-science science stable
# sources.list.d/pointless.list
deb https://its-pointless.github.io/files/24 termux extras
# x11-repo (sources.list.d/x11.list)
deb https://packages.termux.org/apt/termux-x11 x11 main
Updatable packages:
All packages up to date
Android version:
11
Kernel build information:
Linux localhost 4.14.212-gccfcc225a50c-ab7508431 #1 SMP PREEMPT Wed Jun 30 09:53:57 UTC 2021 aarch64 Android
Device manufacturer:
Google
Device model:
Pixel 4a
```

**Comment by easyaspi314** [comment]
Still not fixed.

**Comment by Grimler91** [comment]
Should be fixed in 0.7.1-2, please test it

**Comment by easyaspi314** [comment]
Yes, that works, thanks 😊
